### PR TITLE
.sav parsing: JetAIUpdate and Drawables without GameObjects

### PR DESCRIPTION
**Pull request by Skleni**

### CODE REVIEW

File: src/OpenSage.Game.Tests/Data/Sav/SaveFileTests.cs [hunk 1]
```diff
@@ -43,6 +43,20 @@ public void CanLoadSaveFiles(string relativePath)
             game.EndGame();
         }
 
+        [Fact]
+        public void CanLoadTestWeapons()
```
Comment (tgjones) [inline src/OpenSage.Game.Tests/Data/Sav/SaveFileTests.cs:47]: Is it necessary to add this test? I think it should be covered by `CanLoadSaveFiles` above.

**Review by tgjones — APPROVED** [review]
This is awesome!

File: src/OpenSage.Game.Tests/Data/Sav/SaveFileTests.cs (hunk 1 above)
Comment (Skleni) [reply src/OpenSage.Game.Tests/Data/Sav/SaveFileTests.cs:47]: Right, I added it because the other ones were disabled, but I guess it can be removed.

Comment (tgjones) [reply src/OpenSage.Game.Tests/Data/Sav/SaveFileTests.cs]: Ah yes, locally I have this change:

``` csharp
//[Theory(Skip = "Doesn't work yet")]
[Theory]
[MemberData(nameof(GetSaveFiles))]
public void CanLoadSaveFiles(string relativePath)
```

Comment (tgjones) [reply src/OpenSage.Game.Tests/Data/Sav/SaveFileTests.cs]: I think we can enable those tests by default soon - for me locally, there are only 3 failing ones, which happen to be failing on a road-related issue, not a .sav parsing issue ;-)

**Pull request merged**